src/config/field-overrides/programRule.js

```diff
@@ -15,33 +15,50 @@ import { ExpressionStatus } from './program-indicator/ExpressionStatusIcon';
 import { AnalyticsPeriodBoundaries } from './program-indicator/AnalyticsPeriodBoundaries';
 import createExpressionValidator from './program-indicator/createExpressionValidator';
 
+const setValidatorSubscription = (context, subFunc, programId) => {
+    const path = `programRules/condition/description?programId=${
+        programId
+    }`;
+    const { status$, validate } = createExpressionValidator(path, subFunc);
+    context.validatorStatusSubscription = status$.subscribe(subFunc)
```

Comment:
I know this is a bit weird, but since it uses recompose we can't really have methods... Didn't want to migrate this to a class right now before freeze.